src/core/public/core_system.ts

```diff
@@ -168,12 +179,27 @@ export class CoreSystem {
     });
 
     const timing = this.getLoadMarksInfo();
+
+    const navigatorExt = navigator as ExtendedNavigator;
+    const navigatorInfo: Record<string, string> = {};
+    if (navigatorExt.connection?.effectiveType) {
+      navigatorInfo.effectiveNetworkType = navigatorExt.connection.effectiveType;
```

Comment:
When testing `navigator.connection` locally, it provides far from realistic information, and I don't feel it would serve us any purpose other than to misrepresent the user's connection down the road. I think connection speed is extremely valuable, but getting that will be a larger effort (measuring the download speed of existing assets).